src/main/java/com/scalar/db/storage/jdbc/JdbcService.java

```diff
@@ -108,6 +109,40 @@ public Scanner scan(
         resultSet);
   }
 
+  public List<Result> scan(
+      Scan scan, Connection connection, Optional<String> namespace, Optional<String> tableName)
+      throws SQLException {
+    Utility.setTargetToIfNot(scan, namespacePrefix, namespace, tableName);
```

Comment:
I think `getScanner` and `scan` are pretty redundant, so should `scan` use `getScanner` (maybe it's not possible)  or should both delegate the common procedures to the same method?

Reply:
Thank you for reviewing this! 

Sure, I will try that.